lib/rspec/core/output_wrapper.rb

```diff
@@ -3,13 +3,17 @@ module Core
     # @private
     class OutputWrapper
       # @private
-      attr_writer :output
+      attr_accessor :output
 
       # @private
       def initialize(output)
         @output = output
       end
 
+      def respond_to?(name, priv=false)
+        @output.respond_to?(name, priv)
+      end
```

Comment:
The problem is the inverse case, because this may be a string it will say it responds to puts when it doesn't.